### PR TITLE
[IMP] l10n_pe: Update current Peruvian modules and description

### PR DESCRIPTION
**Pull request by gmz-odoo**
1. **Remove** older ‘Introduction section’
2. **Add a table** listing and update of the currently supported features and modules with a description of its functionality (like the format/structure displayed for [Chile Localization](https://www.odoo.com/documentation/17.0/applications/finance/fiscal_localizations/chile.html) page) 
3. **Update** the description of suggested and related resources, such as Smart Tutorial videos
4. Update the reference for current Bank sync options
5. Update image for current listed Bank sync options
6. Correct the tittle of the exporting invoicing section

Expected result and comparison of actual info:
https://docs.google.com/document/d/1uyB_2sKIbab26LHO8mx9ikPOUukN6-DjHUz0jkcgXx0/edit

### CODE REVIEW

**Review by samueljlieber — APPROVED** [review]
Hi @gmz-odoo, thanks for your work on this PR (and on #10563 & #10635)! 

These changes look good to me, approving :)

File: content/applications/finance/fiscal_localizations/peru.rst [hunk 1]
```diff
@@ -2,33 +2,77 @@
...
+
+       - 3rd set of financial reports:
+
+         - PLE 1.1 Libro de Caja
+         - PLE 1.2 Libro de Bancos
+
+   * - :guilabel:`Peruvian - Electronic Delivery Note`
+     - `l10n_pe_edi_stock`
+     - The delivery guide (Guía de Remisión) is needed as proof that you are sending goods between A
+       and B. It is only when a delivery order is validated that you can create the delivery guide.
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/peru.rst:58]:
```suggestion
     - Adds the delivery guide (Guía de Remisión), which is needed as proof that you are sending
     goods between A and B. It is only when a delivery order is validated that the delivery guide
     can be created.
```
Phrasing this bullet point like this gives it parallel structure to the other bullet points. I also removed an unnecessary "you" from the second sentence. Feel free to check out our [documentation guidelines](https://www.odoo.com/documentation/master/contributing/documentation/content_guidelines.html) for more information.

File: content/applications/finance/fiscal_localizations/peru.rst [hunk 2]
```diff
@@ -2,33 +2,77 @@
...
-  Electronic Invoice, based on the SUNAT specification to create and process valid electronic
-  documents, for more technical detail you can access the
-  `SUNAT EDI specifications <https://cpe.sunat.gob.pe/node/88/>`_,
-  that keeps track of new changes and updates.
-  The features of this module are based on the resolutions published on the
-  `SUNAT Legislation <https://www.sunat.gob.pe/legislacion/general/index.html/>`_.
+.. note::
+   - Odoo automatically installs the appropriate package for the company according to the country
+     selected at the creation of the database.
+   - The *Peruvian - Electronic Delivery Guide* module depends on the *Inventory* application to be installed.
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/peru.rst:71]:
```suggestion
   - The *Peruvian - Electronic Delivery Guide* module depends on the **Inventory** application to
   be installed.
```

File: content/applications/finance/fiscal_localizations/peru.rst [hunk 3]
```diff
@@ -236,12 +280,12 @@ password.
...
 ~~~~~~~~~~~~~
 
-The official currency exchange rate in Peru is provided by the Bank of Peru. Odoo can connect
-directly to its services and get the currency rate either automatically or manually.
+The official currency exchange rate in Peru is provided by the SUNAT. Odoo can connect directly to
+its services and get the currency rate either automatically or manually.
 
-.. image:: peru/peru-multicurrency.png
+.. image:: peru/l10n-pe-banksync-sunat.png
    :align: center
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/peru.rst:287]: Our content guidelines have been updated and we no longer add any alignment to screenshots

File: content/applications/finance/fiscal_localizations/peru.rst [hunk 4]
```diff
@@ -489,8 +533,8 @@ The price list in the IAP is always displayed in EUR.
 Special Use cases
 ~~~~~~~~~~~~~~~~~
 
-Cancellation process
-********************
+Export invoices
```
Comment (dade-odoo) [inline content/applications/finance/fiscal_localizations/peru.rst:536]: This section does seem like it's about the cancellation process and not about exporting invoices. Are you sure you didn't mean to change the title of the other "Cancellation process" section after the credit consumption warning block?

**Review by dade-odoo — CHANGES_REQUESTED** [review]
Hey again @gmz-odoo thanks for the addition to the Peruvian localization! I've got a few more comments and suggestions here - please let me know if you have any questions on them :)

File: content/applications/finance/fiscal_localizations/peru.rst [hunk 5]
```diff
@@ -2,33 +2,77 @@
...
+:ref:`Install <general/install>` the following modules to utilize all the current features of the
+Peruvian localization.
+
+.. list-table::
+   :header-rows: 1
+
+   * - Name
+     - Technical name
+     - Description
+   * - :guilabel:`Chile - Accounting`
```
[the remark this answers is not in the document]
Comment (gmz-odoo) [reply content/applications/finance/fiscal_localizations/peru.rst:23]: Ooopsie, Peru indeed, thanks!

File: content/applications/finance/fiscal_localizations/peru.rst (hunk 1 above)
Comment (gmz-odoo) [reply content/applications/finance/fiscal_localizations/peru.rst:58]: Wonderful :1st_place_medal: thanks @dade-odoo

File: content/applications/finance/fiscal_localizations/peru.rst (hunk 4 above)
Comment (gmz-odoo) [reply content/applications/finance/fiscal_localizations/peru.rst:536]: You've hit the nail on the head :100: 

@dade-odoo  It is indeed, that section after the One Credit warning block, THANKS d:

image
image

**Comment by samueljlieber** [comment]
Thank you @dade-odoo and @gmz-odoo! Feedback implemented in https://github.com/odoo/documentation/commit/2f966ca2cc6ec54abd8adfa9f0c265d0b5f8a34d!

**Review by dade-odoo — APPROVED** [review]
Thanks for making those changes @gmz-odoo and @samueljlieber ! All good for me, I'll pass you on to the final review :)

**Review by xpl-odoo — APPROVED** [review]
@robodoo delegate=gmz-odoo,samueljlieber

**Comment by samueljlieber** [comment]
@robodoo r+

**Pull request closed without merging**